hack/generate_template.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+import oyaml as yaml
+
+import os
+import sys
+import argparse
+import copy
+
+def get_yaml_all(filename):
+    with open(filename,'r') as input_file:
+        return list(yaml.safe_load_all(input_file))
+
+def get_yaml(filename):
+    with open(filename,'r') as input_file:
+        return yaml.safe_load(input_file)
+
+def get_all_yaml_files(path):
+    file_paths = []
+    for r,d,f in os.walk(path):
+        for file in f:
+            if file.endswith('.yml') or file.endswith('.yaml'):
+                file_paths.append(os.path.join(r,file))
+        # break, so we don't recurse
+        break
+    file_paths = sorted(file_paths)
+    return file_paths
+
+def get_all_yaml_obj(file_paths):
+    yaml_objs = []
+    for file in file_paths:
+        objects = get_yaml_all(file)
+        for obj in objects:
+            yaml_objs.append(obj)
+    return yaml_objs
+
+def process_yamls(name, directory, obj):
+    o = copy.deepcopy(obj)
+    # Get all yaml files as array of yaml objects
+    yamls = get_all_yaml_obj(get_all_yaml_files(directory))
+    if len(yamls) == 0:
+        return
+
+    # Find SA name.  It will be used for processing.
+    sa_name = ""
+    for y in yamls:
+        if y['kind'] == 'ServiceAccount':
+            sa_name = y['metadata']['name']
+            break
+
+    # Find all Roles bound to the SA where the subject is in the same NS as the SA.
+    # These Roles are managed by CSV only.
+    sa_role_names = []
+    for y in yamls:
+        if y['kind'] == 'RoleBinding':
+            if 'namespace' not in y['metadata']:
+                print("RoleBinding '{}' is missing .metadata.namespace".format(y['metadata']['name']))
+            for s in y['subjects']:
+                if 'namespace' not in s:
+                    print("RoleBinding '{}' is missing .subjects[].namespace".format(y['metadata']['name']))
+                if y['roleRef']['kind'] == "Role" and s['kind'] == 'ServiceAccount' and y['metadata']['namespace'] == s['namespace']:
+                    sa_role_names.append(y['roleRef']['name'])
```

Comment:
Is an extra role needed by this operator? They are managed by OLM otherwise

Reply:
If they are managed by old, they should all be here

Reply:
Managed by old? What does that mean?

Reply:
Typo from iOS 
Old == OLM